### PR TITLE
Faq added for GraphQL

### PR DESCRIPTION
**Pull request by fulyauluturk**

### CODE REVIEW

File: docs/synthetics/faq.md [hunk 1]
```diff
@@ -165,6 +165,17 @@ The *Cache-Control* HTTP header was introduced in HTTP 1.1. Systems using HTTP 1
 ```javascript
 'Pragma: no-cache'
 ```
+### Can I call GraphQL endpoints?
+Yes. You can call GraphQL endpoints with [HTTP Monitors](https://sematext.com/docs/synthetics/http-monitor/). In `General` tab pick the `Method Type` and enter the GraphQL endpoint. Set `Content-Type` to `application/json` from the Configure Requests --> Headers tab. 
+
+To send a `GET` request, set the query parameter from the `Query Params` tab.
+
+![GraphQL GET](../images/synthetics/synthetics-faq-graphql-get.png)
```
Comment (gorkemmeydan) [inline docs/synthetics/faq.md:173]: I could not find the related image in `images` folder, could you also double check it?

File: docs/synthetics/faq.md [hunk 2]
```diff
@@ -165,6 +165,17 @@ The *Cache-Control* HTTP header was introduced in HTTP 1.1. Systems using HTTP 1
...
+
+To send a `POST` request, open the `Body` tab, set the content type to `JSON` and enter the query.
+
+![GraphQL POST](../images/synthetics/synthetics-faq-graphql-post.png)
```
Comment (gorkemmeydan) [inline docs/synthetics/faq.md:177]: Possible image link missing.

Comment (fulyauluturk) [reply docs/synthetics/faq.md:177]: The images were directly committed to master.

**Review by gorkemmeydan — APPROVED** [review]

**Review by cpipilas — APPROVED** [review]
LGTM (no tech review)

**Pull request merged**